### ISSUE TITLE
android：Attempt to invoke virtual method 'double java.lang.Double.doubleValue()' on a null object reference

### ISSUE THREAD

**Issue by daltachange** [react-native-camera/react-native-camera#3435]
![3961662618770_ pic](https://user-images.githubusercontent.com/12230238/189051175-cb8e1350-2f60-4051-862a-61565358e12f.jpg)

version：4.2.1

code:

  useEffect(() => {
    const focusScreenListener = props.navigation.addListener('focus', () => {
      if (cameraRef.current) {
        cameraRef.current.resumePreview(); // error 
      }
    });
    return () => {
      props.navigation.removeListener('focus', focusScreenListener);
    };
    // eslint-disable-next-line react-hooks/exhaustive-deps
  }, []);

**Comment by olivertylsar** [comment]
Yes, same problem here. Everything is fine until I call `pausePreview` or `resumePreview` in an `useEffect` callback. 

Any updates on this?
